src/EpiEasyEvents/DisabledEventsHandlerScopeConfiguration.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Forte.EpiEasyEvents
+{
+    internal static class DisabledEventsHandlerScopeConfiguration
```

Comment:
Just a suggestion to make this class name more general.
1. It looks a little bit odd to see `Disabled` twice: 'Disabled(...).Is(...)Disabled'
2. In case of potential new scope option one would have to only create new property here instead of creating new class/renaming this one then

Reply:
Fixed in 4f88405e97a5a8f02bb1bfc62c02ba033432d786